source/features/tags-dropdown.tsx

```diff
@@ -23,22 +23,29 @@ function init(): false | void {
 					role="menu"
 					style={{zIndex: 99}}
 				>
-					<include-fragment className="select-menu-loading-overlay anim-pulse" onLoad={onFragmentLoaded}>
+					<include-fragment className="select-menu-loading-overlay anim-pulse" onLoad={changeTabToTags}>
 						{octofaceIcon()}
 					</include-fragment>
 				</details-menu>
 			</details>
 		</div>
 	);
+
+	// https://github.com/github/remote-input-element#events
+	// Wait until the network request is finished and HTML body is updated
+	// "remote-input-success" event is bubbled
+	select('.rgh-tags-dropdown')!.addEventListener('remote-input-success', updateLinksToTag);
```

Comment:
Support for `on-remote-input-success` JSX attributes [has been added](https://github.com/vadimdemedes/dom-chef/pull/71), in case you want to move this.